tests/functional/adapter/test_table_type.py

```diff
@@ -295,3 +296,22 @@ def models(self):
             "incremental_test_model.sql": insertoverwrite_iceberg_sql,
             "schema.yml": schema_base_yml,
         }
+
+
+# Testing iceberg format version 1
+
+
+def replace_with_iceberg_v1(model):
+    # Add iceberg table_type as a config parameter for existing tests
+    pattern = r"(config\s*\([^)]*\))"
+    result = re.sub(pattern, lambda x: x.group(0).rstrip(")") + ", format_version=1)", model)
```

Comment:
In hue or hive, What happens if you don't provide the format_version for the iceberg table? Ideally should we not override the default behavior.

Reply:
Made default format_version as v1. Hive also creates a table with v1 spec if we don't specify format-version.